applications/mne_analyze/libs/anShared/Management/eventmanager.h

```diff
@@ -190,13 +191,16 @@ class ANSHAREDSHARED_EXPORT EventManager : public QThread
      */
     void run() override;
 
-    QMultiMap<EVENT_TYPE, Communicator*> m_routingTable;    /**< Map that holds routing information */
-    QQueue<QSharedPointer<Event> > m_eventQ;                /**< Queue that buffers all published events */
+    QMultiMap<EVENT_TYPE, Communicator*>    m_routingTable;         /**< Map that holds routing information */
+    QQueue<QSharedPointer<Event> >          m_eventQ;               /**< Queue that buffers all published events */
 
-    QMutex m_eventQMutex;                                   /**< Guarding mutex for the event queue */
-    QMutex m_routingTableMutex;                             /**< Guarding mutex for the routing table */
-    volatile long m_sleepTime;                              /**< Controlling execution frequency of main loop */
-    volatile bool m_running;                                /**< Flag for remembering whether the EventManager is currently started or stopped */
+    QMutex                                  m_eventQMutex;          /**< Guarding mutex for the event queue */
```

Comment:
Do we need these two mutexes? are they still used?

Reply:
Yes, m_eventQMutex still guards reading from and writing to the event queue and m_routingTableMutex guards the subscriptions.